github/pulls_reviews.go

```diff
@@ -0,0 +1,14 @@
+package github
+
+import "time"
+
+// No docs are out for this quite yet.
+type PullRequestReview struct {
+	ID          *int       `json:"id,omitempty"`
+	User        *User      `json:"user,omitempty"`
+	Body        *string    `json:"body,omitempty"`
+	SubmittedAt *time.Time `json:"submitted_at,omitempty"`
+
+	// State can be "approved", "rejected", or "commented".
+	State *string `json:"state,omitempty"`
+}
```

Comment:
@shurcooL I put it in its own file in anticipation of the REST API being released (there's an API for nearly every feature on the site). My understanding is that this is where the eventual REST API code will go. I want a struct that is unified between the webhooks and REST API, and `github/event_types.go` is already nearly 550 lines long.